web-common/src/features/alerts/extract-alert-form-values.ts

```diff
@@ -45,7 +45,7 @@ export function extractAlertFormValues(
     queryArgs.dimensions as V1MetricsViewAggregationDimension[];
 
   const timeRange = (queryArgs.timeRange as V1TimeRange) ?? {
-    isoDuration: metricsViewSpec.defaultTimeRange ?? TimeRangePreset.ALL_TIME,
+    isoDuration: TimeRangePreset.ALL_TIME,
```

Comment:
@AdityaHegde Does this look okay to you? The `defaultTimeRange` on the metrics view is deprecated anyway in favour of the explore spec field.

Reply:
Hmmm it was a fallback anyway, so it should be fine to remove it.

Reply:
Great, thought so!